fpm/src/fpm.f90

```diff
@@ -42,8 +43,15 @@ subroutine build_model(model, settings, package, error)
     integer :: i, j
     type(package_config_t) :: dependency
     character(len=:), allocatable :: manifest, lib_dir
+
+    logical :: duplicates_found = .false.
     type(string_t) :: include_dir
 
+    if(settings%verbose)then
+       write(*,*)'<INFO>BUILD_NAME:',settings%build_name
+       write(*,*)'<INFO>COMPILER:  ',settings%compiler
+    endif
+
```

Comment:
This looks like a remnant of the merge from master, and isn't necessary in this PR.
```suggestion
```